samples/node/content-caching.js

```diff
@@ -49,9 +49,7 @@ async function run() {
 
```

Comment:
I guess can you add a TODO or reminder to update line 41 here with the new official model name along with where you're going to do it in the integration test?

Reply:
I just added it in at this point. Done!